packages/sit-onyx/src/components/OnyxAppLayout/OnyxAppLayout.vue

```diff
@@ -0,0 +1,84 @@
+<script lang="ts" setup>
+import { useSlots } from "vue";
+
+const props = withDefaults(
+  defineProps<{
+    /** Whether the nav bar will stick to the left or to the top of the app */
+    navBarAlignment: "top" | "left";
+  }>(),
+  { navBarAlignment: "top" },
+);
+defineSlots<{
+  /** Navigation area of the application */
+  navBar(props: Record<string, never>): unknown;
+  /** Page content area of the application */
+  default(props: Record<string, never>): unknown;
+  /** Overlays that cover the page and exclude the nav area */
+  pageOverlay(props: Record<string, never>): unknown;
+  /** Overlays that cover the complete page */
+  appOverlay(props: Record<string, never>): unknown;
+}>();
+
+const slots = useSlots();
+</script>
+
+<template>
+  <div class="onyx-app" :class="{ 'onyx-app--horizontal': props.navBarAlignment === 'left' }">
+    <nav class="onyx-app__nav">
```

Comment:
```suggestion
    <nav v-if="slots.navBar" class="onyx-app__nav">
```

I could imagine use cases like landing pages where no nav should be displayed